src/Commands/HTTPS/RemoveCommand.php

```diff
@@ -15,17 +15,17 @@ class RemoveCommand extends TerminusCommand implements SiteAwareInterface
     use SiteAwareTrait;
 
     /**
-     * Remove HTTPS from an environment
+     * Disables HTTPS and removes the SSL Certificate from the environment.
```

Comment:
The internet seems half for capitalizing 'Certificate' and half against. I'm against.